From 5105e6afb390f037e6d6a3f14dfe1362f1f677b6 Mon Sep 17 00:00:00 2001
From: Xulin Sun <xulin.sun@windriver.com>
Date: Wed, 3 Jul 2019 15:35:43 +0800
Subject: [PATCH 198/198] dts: bcm2710-rpi-3-b-plus: enable series peripheral
 devices

Some peripheral devices are disabled by default, so open these.

Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 arch/arm/boot/dts/bcm2710-rpi-3-b-plus.dts | 11 +++++++++++
 1 file changed, 11 insertions(+)
 mode change 100644 => 100755 arch/arm/boot/dts/bcm2710-rpi-3-b-plus.dts

diff --git a/arch/arm/boot/dts/bcm2710-rpi-3-b-plus.dts b/arch/arm/boot/dts/bcm2710-rpi-3-b-plus.dts
old mode 100644
new mode 100755
index fa95f4e3653f..20d7f49519db
--- a/arch/arm/boot/dts/bcm2710-rpi-3-b-plus.dts
+++ b/arch/arm/boot/dts/bcm2710-rpi-3-b-plus.dts
@@ -92,6 +92,10 @@
 	};
 };
 
+&fb {
+	status = "okay";
+};
+
 &uart0 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&uart0_pins &bt_pins>;
@@ -108,6 +112,7 @@
 	pinctrl-names = "default";
 	pinctrl-0 = <&spi0_pins &spi0_cs_pins>;
 	cs-gpios = <&gpio 8 1>, <&gpio 7 1>;
+	status = "okay";
 
 	spidev0: spidev@0{
 		compatible = "spidev";
@@ -130,21 +135,25 @@
 	pinctrl-names = "default";
 	pinctrl-0 = <&i2c0_pins>;
 	clock-frequency = <100000>;
+	status = "okay";
 };
 
 &i2c1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&i2c1_pins>;
 	clock-frequency = <100000>;
+	status = "okay";
 };
 
 &i2c2 {
 	clock-frequency = <100000>;
+	status = "okay";
 };
 
 &i2s {
 	pinctrl-names = "default";
 	pinctrl-0 = <&i2s_pins>;
+	status = "okay";
 };
 
 &leds {
@@ -163,11 +172,13 @@
 
 &hdmi {
 	hpd-gpios = <&gpio 28 GPIO_ACTIVE_LOW>;
+	status = "okay";
 };
 
 &audio {
 	pinctrl-names = "default";
 	pinctrl-0 = <&audio_pins>;
+	status = "okay";
 };
 
 / {
-- 
2.17.1

